ra/ra.go

```diff
@@ -653,14 +652,14 @@ func (ra *RegistrationAuthorityImpl) checkInvalidAuthorizationLimit(ctx context.
 	// here.
 	noKey := ""
 	threshold, overrideKey := limit.GetThreshold(noKey, regID)
-	if overrideKey != "" {
-		utilization := float64(count.Count) / float64(threshold)
-		ra.rlOverrideUsageGauge.WithLabelValues(ratelimit.InvalidAuthorizationsPerAccount, overrideKey).Set(utilization)
-	}
 	if count.Count >= threshold {
 		ra.log.Infof("Rate limit exceeded, InvalidAuthorizationsByRegID, regID: %d", regID)
 		return berrors.FailedValidationError(0, "too many failed authorizations recently")
 	}
+	if overrideKey != "" {
+		utilization := float64(count.Count+1) / float64(threshold)
```

Comment:
I think this +1 is also not correct, but for the opposite reason: this NewOrder request won't (on its own) generate _any_ invalid authorizations. We have to wait for them to try to fulfill challenges before that happens.